src/bindings.lisp

```diff
@@ -71,6 +71,15 @@
                                             :function-prefix (api-function-prefix api)
                                             :error-map (api-error-map api))))))))))
 
+(defun format-init-args (initialize-lisp-args)
+  "This formats initialization arguments as either strings or environment variables."
+  (loop :for arg :in initialize-lisp-args
+        :collect (etypecase arg
+                   (symbol
```

Comment:
A very minor security thing would be to check that symbol or string are both alphanumeric. Since this is a string substitution, it's possible to get things wrong. Maybe not worth it for now...